recipes/jupyterlab-nvdashboard/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "jupyterlab-nvdashboard" %}
+{% set version = "0.1.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/rapidsai/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 6cc0d741edf3e9865b8a0cd8eab1c0cb6de131e79001519bb3a1756b165d2617
+
+build:
+  noarch: python
+  number: 0
+  script:
+    - {{ PYTHON }} -m pip install . -vv
+    - jupyter labextension install jupyterlab-nvdashboard@{{ version }}
```

Comment:
We had in other packages problems with this and it might be useful to install this with `--no-build` but sadly there is currently no stable solution to package JupyterLab extensions with conda. If this or a adaption of it works, please report back in https://github.com/conda-forge/conda-forge.github.io/issues/761.